pilot/pkg/config/kube/gateway/backend_policies.go

```diff
@@ -0,0 +1,423 @@
+				tls.CredentialName = "invalid://"
+			} else if len(s.Validation.CACertificateRefs) > 1 {
+				conds[string(gatewayalpha2.PolicyConditionAccepted)].message += "; warning: only the first caCertificateRefs will be used"
+			}
+		}
+		for idx, t := range i.Spec.TargetRefs {
+			conds = maps.Clone(conds)
+			refo, err := references.LocalPolicyTargetRef(t.LocalPolicyTargetReference, i.Namespace)
+			if err == nil {
+				switch refo.(type) {
+				case *v1.Service:
+				default:
+					err = fmt.Errorf("unsupported reference kind: %v", t.Kind)
+				}
+			}
+			if err != nil {
+				conds[string(gatewayalpha2.PolicyConditionAccepted)].error = &ConfigError{
+					Reason:  string(gatewayalpha2.PolicyReasonTargetNotFound),
+					Message: fmt.Sprintf("targetRefs invalid: %v", err),
+				}
+			} else {
+				// Only create an object if we can resolve the target
+				res = append(res, BackendPolicy{
+					Source: TypedNamedspacedName{
+						NamespacedName: config.NamespacedName(i),
+						Kind:           kind.BackendTLSPolicy,
+					},
+					TargetIndex: idx,
+					Target: TypedNamedspacedName{
+						NamespacedName: types.NamespacedName{
+							Name:      string(t.Name),
+							Namespace: i.Namespace,
+						},
+						Kind: kind.MustFromGVK(schematypes.GvkFromObject(refo.(controllers.Object))),
+					},
+					TLS:          tls,
+					CreationTime: i.CreationTimestamp.Time,
+				})
+			}
+			// TODO: section name
+			ancestors = append(ancestors, setAncestorStatus(t.LocalPolicyTargetReference, status, i.Generation, conds))
+		}
+		status.Ancestors = mergeAncestors(status.Ancestors, ancestors)
+		return status, res
+	}, opts.WithName("BackendTLSPolicy")...)
+}
+
+func BackendTrafficPolicyCollection(
+	trafficPolicies krt.Collection[*gatewayx.XBackendTrafficPolicy],
+	references *ReferenceSet,
+	opts krt.OptionsBuilder,
+) (krt.StatusCollection[*gatewayx.XBackendTrafficPolicy, gatewayx.PolicyStatus], krt.Collection[BackendPolicy]) {
+	return krt.NewStatusManyCollection(trafficPolicies, func(ctx krt.HandlerContext, i *gatewayx.XBackendTrafficPolicy) (
+		*gatewayx.PolicyStatus,
+		[]BackendPolicy,
+	) {
+		status := i.Status.DeepCopy()
+		res := make([]BackendPolicy, 0, len(i.Spec.TargetRefs))
+		ancestors := make([]gatewayalpha2.PolicyAncestorStatus, 0, len(i.Spec.TargetRefs))
+
+		lb := &networking.LoadBalancerSettings{}
+
+		conds := map[string]*condition{
+			string(gatewayalpha2.PolicyConditionAccepted): {
+				reason:  string(gatewayalpha2.PolicyReasonAccepted),
+				message: "Configuration is valid",
+			},
+		}
+		var unsupported []string
+		// TODO: implement i.Spec.SessionPersistence. This will need to map into a StatefulSession filter which Istio doesn't
+		// currently support on DestinationRule
+		if i.Spec.SessionPersistence != nil {
+			unsupported = append(unsupported, "sessionPersistence")
+		}
+		// TODO: implement i.Spec.RetryConstraint. This doesn't have upstream Envoy support
```

Comment:
Should we file an issue? I imagine EnvoyGateway will push for this, but would be nice to have a tracking issue in the comment

Reply:
Per @mikemorris https://github.com/kubernetes-sigs/gateway-api/pull/3573#issuecomment-2640799905 @tonya11en will be able to help out here. A tracking issue is good though

Reply:
Tracking this in https://github.com/istio/istio/issues/55838